OsmAnd/src/net/osmand/plus/routepreparationmenu/MapRouteInfoMenu.java

```diff
@@ -194,6 +199,13 @@
 	private boolean addButtonCollapsing;
 	private boolean addButtonCollapsed;
 
+	private List<WorldRegion> suggestedMissingMaps;
+	boolean isNavigationDisable;
```

Comment:
doesn't make sense to disable navigation button cause it same as empty navigation result, because it's done async user can click navigation button faster than anything is detected